site/cypress/fixtures/fragments/feedback.js

```diff
@@ -0,0 +1,8 @@
+module.exports={
+    cancelFeedbackButton: `[data-testid='feedback-screen-cancel-button']`,
+    feedbackButtonText: `[data-testid='large-button']`,
+    feedbackButtonSelected: `[data-testid='large-button-selected-text']`,
+    feedbackTextField: `[data-testid='feedback-content']`,
+    feedbackNotification: `[data-testid='notification-text-content']`,
+    submitFeedbackButton: `[data-testid='feedback-screen-submit-button']`
+}
```

Comment:
Several of these will select other elements on other pages (than the feedback one.) Do we want to extend these queries to something like the 'large button' inside the feedback screen. As opposed to any 'large button'

Reply:
Done.